cmd/telemeter-benchmark/main.go

```diff
@@ -61,6 +62,7 @@ func main() {
 	cmd.Flags().DurationVar(&opt.Interval, "interval", opt.Interval, "The interval between scrapes. Prometheus returns the last 5 minutes of metrics when invoking the federation endpoint.")
 	cmd.Flags().StringVar(&opt.Listen, "listen", opt.Listen, "A host:port to listen on for health and metrics.")
 	cmd.Flags().IntVar(&opt.Workers, "workers", opt.Workers, "The number of workers to run in parallel.")
+	cmd.Flags().Float64Var(&opt.FaultyProbability, "faulty-probability", opt.FaultyProbability, "The probability a worker is a cluster, which sends faulty payloads.")
```

Comment:
This comment needs a tweak. Right now it says “clusters are things that send faulty payloads and this is the probability that a worker is a cluster”.

We want “The pobability that a worker is a cluster that sends faulty payloads.”

Or simpler yet “The probability that a worker sends faulty payloads.”